packages/create-spectacle/package.json

```diff
@@ -35,7 +35,24 @@
     "lint": "wireit",
     "lint:fix": "wireit",
     "prettier": "wireit",
-    "prettier:fix": "wireit"
+    "prettier:fix": "wireit",
+    "examples:clean": "rimraf .examples",
+    "examples:test": "nps jest",
```

Comment:
There's just one test that assumes a final "made with" slide from a dev server running on port 3000. Open to other more flexible suggestions!

Reply:
we could port forward but that feels over engineered here. that to me is just testing if we use `commander` correctly rather than the output correctly building and not having runtime errors.